pt/news/_posts/2018-06-20-suporte-do-ruby-2-2-acabou.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: news_post
+title: "O suporte do Ruby 2.2 acabou"
+author: "antonpaisov"
+translator: "bannergames"
+date: 2018-06-20 00:00:00 +0000
+lang: pt
+---
+
+Anunciamos que todo o suporte das séries 2.2 do Ruby acabou.
+
+Após o lançamento do Ruby 2.2.7 em 28 de março de 2017,
+o suporte das séries 2.2 do Ruby estava em fase de manunteção de segurança.
```

Comment:
**da série**

Reply:
@jcserracampos It says "manunteção", it should be "manutenção".